include/afl-fuzz.h

```diff
@@ -734,6 +739,7 @@ typedef struct afl_state {
   char  mutation[8072];
   char  m_tmp[4096];
   FILE *introspection_file;
+  u32   bitsmap_size;
```

Comment:
`bitmap`, without the s?

Reply:
yes, that is for counting the bits before classifying them. just for introspection compile